events/test.py

```diff
@@ -0,0 +1,16 @@
+import json
+import requests
+
+
+def read_events():
+    with open('sample_data.json', 'r') as f:
+        body = f.read()
+        return json.loads(body)
```

Comment:
Isn't this what [`json.load`](https://docs.python.org/3/library/json.html#json.load) is for?

```python
with open('sample_data.json', 'r') as f:
    return json.load(f)
```